blackbox/docs/release_notes/2.1.9.txt

```diff
@@ -0,0 +1,65 @@
+.. _version_2.1.9:
+
+=============
+Version 2.1.9
+=============
+
+Released on 2017/09/08.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.1.3` or higher before you upgrade to 2.1.9.
+
+   If you want to perform a :ref:`rolling_upgrade`, your current CrateDB
+   version number must be :ref:`version_2.1.0`.  If you want to upgrade from a
+   version prior to this, the upgrade will introduce all of the breaking
+   changes listed for :ref:`version_2.1.0`, and will require a
+   :ref:`full_restart_upgrade`.
+
+   Consult the :ref:`version_2.1.0_upgrade_notes` for ``2.1.0`` when upgrading.
+
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Changelog
+=========
+
+Fixes
+-----
+
+- Admin UI improvements.
```

Comment:
Didn't we want to make them explicit in this list? Copy them over from the Admin UI release?